atlasdb-client/src/main/java/com/palantir/atlasdb/stream/GenericStreamStore.java

```diff
@@ -31,7 +32,17 @@
 
     Map<Sha256Hash, ID> lookupStreamIdsByHash(Transaction t, final Set<Sha256Hash> hashes);
 
+    /**
+     * @deprecated use #loadSingleStream instead.
```

Comment:
We aren't committing to a specific time frame yet. We don't want to just remove or replace it, because that would be a compile-breaking change (albeit maybe a rather simple one?).